docs/user-guide/configuring-http-api.rst

```diff
@@ -1,21 +1,29 @@
 :desc: Find out how to use Rasa's HTTP API to integrate Rasa
        with your backend components.
 
-.. _running-the-server:
+.. _configuring-http-api:
 
-Running the Server
-==================
+Configuring HTTP API
+=====================
 
 .. edit-link::
 
 .. contents::
    :local:
 
-Running the HTTP server
------------------------
+Using Rasa's HTTP API
+--------------------------
+
+.. note:: 
+
+    Running a model locally as described below is not a scalable approach to 
+    serving your assistant to users. If you want to deploy your assistant,
+    see `how-to-deploy`_. The instructions below are relevant for  
+    configuring how a model is run within a Docker container or for testing 
+    the HTTP API locally.
```

Comment:
The instructions below are relevant for configuring how a model is run within a Docker container or for testing the HTTP API locally. If you want to deploy your assistant to users, see :ref:`how-to-deploy`

otherwise sounds too aggressive